ui/src/organizations/components/CreateVariableOverlay.tsx

```diff
@@ -0,0 +1,119 @@
+// Libraries
+import React, {PureComponent, ChangeEvent} from 'react'
+
+// Styles
+import 'src/organizations/components/CreateVariableOverlay.scss'
+
+// Components
+import {
+  Form,
+  OverlayBody,
+  OverlayHeading,
+  OverlayContainer,
+  Input,
+  Button,
+  ComponentColor,
+  ComponentStatus,
+  ButtonType,
+  OverlayFooter,
+} from 'src/clockface'
+import FluxEditor from 'src/shared/components/FluxEditor'
+
+interface Props {
+  onCreateVariable: () => void
+  onCloseModal: () => void
+}
+
+interface State {
+  name: string
+  script: string
+  nameInputStatus: ComponentStatus
+  errorMessage: string
+}
+
+export default class CreateOrgOverlay extends PureComponent<Props, State> {
+  constructor(props) {
+    super(props)
+    this.state = {
+      name: '',
+      script: '',
+      nameInputStatus: ComponentStatus.Default,
+      errorMessage: '',
+    }
+  }
+
+  public render() {
+    const {onCloseModal} = this.props
+    const {nameInputStatus, name, script} = this.state
+
+    return (
+      <OverlayContainer maxWidth={1000}>
+        <OverlayHeading
+          title="Create Variable"
+          onDismiss={this.props.onCloseModal}
+        />
+
+        <Form>
+          <OverlayBody>
+            <div className="overlay-flux-editor--spacing">
+              <Form.Element label="Name">
+                <Input
+                  placeholder="Give your variable a name"
+                  name="name"
+                  autoFocus={true}
+                  value={name}
+                  onChange={this.handleChangeInput}
+                  status={nameInputStatus}
+                />
+              </Form.Element>
+            </div>
+
+            <Form.Element label="Value">
+              <div className="overlay-flux-editor">
+                <FluxEditor
+                  script={script}
+                  onChangeScript={this.handleChangeScript}
+                  visibility="visible"
+                  status={{text: '', type: ''}}
```

Comment:
Maybe there could be some kind default status prop in the `FluxEditor` so we don't have to specify this when we're not updating it

Reply:
good idea 👍